plugins/wysiwygarea/plugin.js

```diff
@@ -95,6 +95,40 @@
 
 					editor.editable( new framedWysiwyg( editor, iframe.$.contentWindow.document.body ) );
 					editor.setData( editor.getData( 1 ), callback );
+
+					iframe.$.onloadFromSetData = false;
+
+					editor.on( 'mode', backupOnLoad, { iframe: iframe, editor: editor, callback: callback } );
+				}
+
+				function backupOnLoad( evt ) {
+					evt && evt.removeListener();
+					var iframe = this.iframe;
+					var editor = this.editor;
+					var callback = this.callback;
+
+					iframe.on( 'load', function( evt ) {
+						if ( iframe.$.onloadFromSetData ) {
+							iframe.$.onloadFromSetData = false;
+							return;
+						}
+
+						var cacheData = editor.getData( false );
+
+						// Remove current editable, but preserve iframe
+						editor.editable().saveIframe = true;
+						editor.editable( 0 );
```

Comment:
`editor.editable()` takes an element as an argument and while it works with `0`, it would be more fitting and intuitive to pass `null` here.